### PR TITLE
review

### PR DESCRIPTION
**Pull request by Eriner**
I split out the tags into a separate job. Something about adding a CI step from "crazy-max" into the primary build pipeline... (:  The tags are captured from the output. CI change is untested.

Still a TODO re: the ports, I figure you are more familiar with this than I.

Tag: https://github.com/SiaFoundation/siad/pull/2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,27 +6,32 @@ RUN apk update && \
...
 
 WORKDIR /app
-
 COPY . .
-
 ENV GOBIN=/app/bin
 
 # need to run git status first to fix GIT_DIRTY detection in makefile
 RUN git status > /dev/null && \
+        update-ca-certificates && \
```
Comment (n8maninger) [inline Dockerfile:14]: `update-ca-certificates` is run as part of the apk line, unnecessary

File: Dockerfile [hunk 2]
```diff
@@ -6,27 +6,32 @@ RUN apk update && \
...
 EXPOSE 9981 9982 9983 9984
 
 # SIA_WALLET_PASSWORD is used to automatically unlock the wallet
 ENV SIA_WALLET_PASSWORD=
 # SIA_API_PASSWORD sets the password used for API authentication
 ENV SIA_API_PASSWORD=
 
+# The diretive below is not functionality supported by siad, but it
+# would be nice if at some point it was. :)
+#HEALTHCHECK --interval=5m --timeout=10s CMD [ "/siad", "--health" ]
```
Comment (n8maninger) [inline Dockerfile:33]: no need to add it since it's unsupported, it can always be added in the future without a placeholder

File: Dockerfile [hunk 3]
```diff
@@ -6,27 +6,32 @@ RUN apk update && \
...
 	make static
 
 FROM scratch
 
 COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 COPY --from=build /app/release /
 
+# TODO: can we list these with their protocols (/tcp or /udp)?
+# It would also be nice to document what these are.
 EXPOSE 9981 9982 9983 9984
```
Comment (n8maninger) [inline Dockerfile:24]: all tcp, will document

**Comment by n8maninger** [comment]
I was debating removing crazymax and building something similar, I'm not particularly happy with it either.

File: .github/workflows/publish-docker.yml [hunk 4]
```diff
@@ -9,8 +9,23 @@ on:
       - 'v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+'
 
 jobs:
+  get-tags:
```
Comment (n8maninger) [inline .github/workflows/publish-docker.yml:12]: What's the benefit of splitting this out from the other CI steps?

**Comment by Eriner** [comment]
I don't mind using it, just as long as its not in the primary build path. Getting the tags in Actions is very frustrating. There are a few ways of doing it, but all of them are annoying. I don't have a problem with it as modified in this PR.

Feel free to trash the `HEALTHCHECK`.

**Pull request closed without merging**